doc/docs.md

```diff
@@ -180,7 +180,7 @@ by using any of the following commands in a terminal:
     * [Structs with reference fields](#structs-with-reference-fields)
     * [sizeof and __offsetof](#sizeof-and-__offsetof)
     * [Limited operator overloading](#limited-operator-overloading)
-    * [Performance tuning](#performance-tuning)
+    * [Tuning](#tuning)
```

Comment:
Please do not break links without need. The old section name was fine.

Reply:
Doesn't new name suit better for this section since it describes not only performance tweaks, but also memory usage and executable size tweaks?

Reply:
Aren't those all related to performance?  In a way, at least.  Hard to get something to perform better (or even correctly), if it doesn't fit in memory, and executable size directly affects memory as well.

Reply:
> Doesn't new name suit better for this section since it describes not only performance tweaks, but also memory usage and executable size tweaks?

No, that is part of the problem. `Tuning` is very general. In a sense, every option passed to the compiler can be considered to be tuning. For example `-g` or `-gc none` or `-autofree` or `-trace-calls` or `-compress` or `-profile`, will all affect the produced executable, but their main goal is not improving performance.